src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -0,0 +1,1161 @@
+        },
+        {
+          title: 'Create an eCommerce Store with Next.js and Stripe Checkout',
+          byline: 'Colby Fayock・1h 4m・Course',
+          image:
+            'https://d2eip9sf3oo6c2.cloudfront.net/playlists/square_covers/000/412/781/thumb/ecommerce-stripe-next.png',
+          path: '/playlists/create-an-ecommerce-store-with-next-js-and-stripe-checkout-562c',
+          slug: 'create-an-ecommerce-store-with-next-js-and-stripe-checkout-562c',
+          description: `This is a practical project based look at building a working e-commerce store
+        using modern tools and APIs. Excellent for a weekend side-project for your [developer project portfolio](https://joelhooks.com/developer-portfolio)`,
+        },
+        {
+          title: 'Practical Git for Everyday Professional Use',
+          byline: 'Trevor Miller・1h・Course',
+          image:
+            'https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/050/thumb/egghead-practical-git-course.png',
+          path: '/courses/practical-git-for-everyday-professional-use',
+          slug: 'practical-git-for-everyday-professional-use',
+          description: `[git](/q/git) is a critical component in the modern web developers tool box. This course
+         is a solid introduction and goes beyond the basics with some more advanced git commands
+         you are sure to find useful.`,
+        },
+        {
+          title: 'Build an App with the AWS Cloud Development Kit',
+          byline: 'Tomasz Łakomy・1h 4m・Course',
+          image:
+            'https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/450/thumb/EGH_AWS-TS.png',
+          path: '/courses/build-an-app-with-the-aws-cloud-development-kit',
+          slug: 'build-an-app-with-the-aws-cloud-development-kit',
+          description:
+            "Tomasz Łakomy will guide you through using TypeScript to complete the lifecycle of an application powered by AWS CDK. You'll see how to start a project, develop it locally, deploy it globally, then tear it all down when you're done. Excellent kick start for your next side project or your developer portfolio.",
+        },
+      ].filter((resource) => {
+        return resource.slug !== course.slug
+      }),
+      3,
+    )
+
+    // Manually slicing lessons into modules
+    const lessonModules = [
+      {title: 'Project Setup', lessons: lessons.slice(0, 3)},
+      {title: 'Tags', lessons: lessons.slice(3, 7)},
+      {title: 'Variables and Constants', lessons: lessons.slice(7, 15)},
+      {title: 'Conditionals', lessons: lessons.slice(15, 21)},
+      {title: 'Arrays and Loops', lessons: lessons.slice(21, 29)},
+      {title: 'Functions', lessons: lessons.slice(29, 37)},
+      {title: 'Classes', lessons: lessons.slice(37, 46)},
+    ]
```

Comment:
hard coded modules list, we will expect this type of data to come through sanity. We could hard code this in the groq query for this experiment but doing it in the component felt better